### PR TITLE
Fix homepage allignment

### PR DESCRIPTION
**Pull request by pkretzschmar**
Ref #385﻿

### CODE REVIEW

File: src/components/views/homepage/HomePurpleSection.tsx [hunk 1]
```diff
@@ -187,6 +187,7 @@ const ArcSmall = styled(Arc)`
 const ForMakersGivers = styled(Row)`
 	margin-top: 107px;
 	margin-bottom: 50px;
+	justify-content: space-between;
```
Comment (RamRamez) [inline src/components/views/homepage/HomePurpleSection.tsx:190]: there is too much space between them. maybe it's a better idea too set a gap between the two sections instead of `space-between`

image

File: src/components/views/homepage/HomeChangeMakers.tsx [hunk 2]
```diff
@@ -64,12 +64,14 @@ const HomeChangeMakers = () => {
 			<PurpleArc />
 			<CyanArc />
 			<MustardArc />
-			<D3>Calling all Changemakers!</D3>
-			<br />
-			<Lead>
-				Do you have a for-good project that&apos;s creating value for
-				society, for the environment, or for the world?
-			</Lead>
+			<TitleContainer>
```
Comment (RamRamez) [inline src/components/views/homepage/HomeChangeMakers.tsx:67]: why did you add this?

**Review by RamRamez — CHANGES_REQUESTED** [review]

File: src/components/views/homepage/HomeChangeMakers.tsx (hunk 2 above)
Comment (pkretzschmar) [reply src/components/views/homepage/HomeChangeMakers.tsx:67]: Sorry, I forgot to remove, I was just testing some stuff.

File: src/components/views/homepage/HomePurpleSection.tsx [hunk 3]
```diff
@@ -200,7 +201,7 @@ const ForMakersContainers = styled(Col)`
 
 const ForMakersButton = styled(Button)`
 	width: 300px;
-	margin-top: 24px;
+	margin-top: auto;
```
Comment (RamRamez) [inline src/components/views/homepage/HomePurpleSection.tsx]: It's not true according to figma:

image

**Review by RamRamez — APPROVED** [review]
